pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -634,18 +634,6 @@ let
         };
       };
 
-      coenraads.bracket-pair-colorizer-2 = buildVscodeMarketplaceExtension {
```

Comment:
this is also missing an alias

Reply:
Sorry Sandro. I'm no longer active maintainer here.